openapi/paths/storefront/transactions@{id}@{token}@continue.yaml

```
@@ -0,0 +1,27 @@
+parameters:
+  - $ref: ../../components/parameters/resourceId.yaml
+  - $ref: ../../components/parameters/token.yaml
+post:
+  x-products:
+    - Storefront
+  tags:
+    - Transactions
+  summary: Continue processing transaction
+  operationId: StorefrontPostKycRequestContinue
+  x-sdk-operation-name: continueKyc
+  security:
+    - CustomerJWT: []
+  description: Continues processing transaction with specified ID, after all documents of KYC request with specified token have been uploaded.
```

suggestion: It may be good to split this into two sentences to improve readability

```suggestion
  description: |-
     Continues processing a KYC interrupted transaction with a specified ID.
     Use this operation to continue processing the transaction after all documents of the KYC request with specified token have been uploaded.
```